src/test/regress/multi_1_schedule

```diff
@@ -33,6 +33,7 @@ test: ref_citus_local_fkeys
 test: alter_database_owner
 test: distributed_triggers
 test: create_null_dist_key
+test: alter_table_null_dist_key
```

Comment:
Usually we place the tests that require add / remove coordinator step but this test is not one of them. And given that this test is a short one, can we parallelize this with another in this schedule?